src/WaveClip.cpp

```diff
@@ -140,7 +140,9 @@ WaveClip::WaveClip(const WaveClip& orig,
    // current sample block factory, because we might be copying
    // from one project to another
 
-   mOffset = orig.mOffset;
+   mSequenceOffset = orig.mSequenceOffset;
```

Comment:
I observe that this second commit in the draft PR does not build.

In the final PR, that should be corrected, and I would like this internals commit to be first in the sequence.  Or it might be a PR by itself.